### PR TITLE
feat (api): builtin project gpg key

### PR DESCRIPTION
**Pull request by fsamin**
+Fix private key in db

### CODE REVIEW

File: engine/api/environment/dao_key.go [hunk 1]
```diff
@@ -40,6 +41,11 @@ func LoadAllKeys(db gorp.SqlExecutor, env *sdk.Environment) error {
 	for i := range res {
 		p := res[i]
 		keys[i] = sdk.EnvironmentKey(p)
+		decrypted, err := secret.Decrypt([]byte(keys[i].Private))
+		if err != nil {
+			log.Error("LoadAllKeys> Unable to decrypt private key %s/%s", env.Name, keys[i].Name)
```
Comment (yesnault) [inline engine/api/environment/dao_key.go]: missing err in log.Error

File: engine/api/project/dao_key.go [hunk 2]
```diff
@@ -40,6 +41,11 @@ func LoadAllKeys(db gorp.SqlExecutor, proj *sdk.Project) error {
 	for i := range res {
 		p := res[i]
 		keys[i] = sdk.ProjectKey(p)
+		decrypted, err := secret.Decrypt([]byte(keys[i].Private))
+		if err != nil {
+			log.Error("LoadAllKeys> Unable to decrypt private key %s/%s", proj.Key, keys[i].Name)
```
Comment (yesnault) [inline engine/api/project/dao_key.go]: missing err ?

File: engine/api/project/dao.go [hunk 3]
```diff
@@ -128,6 +129,25 @@ func Insert(db gorp.SqlExecutor, store cache.Store, proj *sdk.Project, u *sdk.Us
 		return err
 	}
 	*proj = sdk.Project(dbProj)
+
+	keyID, publicKey, privateKey, err := keys.GeneratePGPKeyPair("builtin")
```
Comment (yesnault) [inline engine/api/project/dao.go]: "builtin" it used 4x, a const can be useful.

File: engine/api/application/dao_key.go [hunk 4]
```diff
@@ -40,6 +41,11 @@ func LoadAllKeys(db gorp.SqlExecutor, app *sdk.Application) error {
 	for i := range res {
 		p := res[i]
 		keys[i] = sdk.ApplicationKey(p)
+		decrypted, err := secret.Decrypt([]byte(keys[i].Private))
+		if err != nil {
+			log.Error("LoadAllKeys> Unable to decrypt private key %s/%s", app.Name, keys[i].Name)
```
Comment (bnjjj) [inline engine/api/application/dao_key.go]: no return ?

**Pull request merged**